lib/util/auth.js

```diff
@@ -5,19 +5,24 @@ import { isOtpMiddleware } from './user'
  * Custom links under the user account dropdown.
  * TODO: Determine/implement the urls below.
  */
-export const accountLinks = [
-  {
-    messageId: 'myAccount',
-    url: ACCOUNT_PATH
-  },
-  {
-    messageId: 'help',
-    // Add a target attribute if you need the link to open in a new window, etc.
-    // (supports the same values as <a target=... >).
-    // target: '_blank',
-    url: '/help'
-  }
-]
+export const accountLinks = (config) => {
```

Comment:
nitpick: can we refactor this to take in `extraMenuItems` instead of the entire config?